extensions/librdkafka/tests/CMakeLists.txt

```diff
@@ -29,10 +29,12 @@ FOREACH(testfile ${KAFKA_TESTS})
     createTests("${testfilename}")
     MATH(EXPR KAFKA_TEST_COUNT "${KAFKA_TEST_COUNT}+1")
     # The line below handles integration test
+    target_include_directories(${testfilename} BEFORE PRIVATE "../..")
```

Comment:
The comment above is outdated, I suggest removing it.

Reply:
I have removed the comment.